tests/Http/Controllers/Api/Entries/View/External/ExportRoutes/ApiRateLimits/RateLimitsMediaExportLocalTest.php

```
@@ -0,0 +1,285 @@
+            $this->assertEntryRowAgainstPayload(
+                $entryRowBundle,
+                $entryPayloads[$i]
+            );
+
+            // Create a fake video file associated with the entry
+            $filename = $videoAnswers[$i]['answer'];
+            $videoContent = MediaGenerator::getFakeVideoContentBase64();
+            Storage::disk('video')->put($this->project->ref . '/' . $filename, $videoContent);
+            // Log the absolute path Laravel actually wrote to
+            $diskRoot = Storage::disk('video')->path($this->project->ref . '/' . $filename);
+            Log::info("File saved to disk at: $diskRoot");
```

PHPMD (2.15.0)

Avoid using static access to class '\Log' in method 'test_rate_limit_api_media'. (Clean Code Rules)

(StaticAccess)